CREATE-NEW-SCALER.md

```diff
@@ -4,7 +4,7 @@
 
 In order to developer a scaler, a developer should do the following:
 1. Download KEDA's code
-2. Define the main pieces of data that you expect the user to supply so the scaler runs properly. For example, if your scaler needs to connect to an external source based on a connection string, you expect the user to supply this connection string in the conifugration within the ScaledObject under `trigger`. This data will be passed to your constructing function as map[string]string.
+2. Define the main pieces of data that you expect the user to supply so the scaler runs properly. For example, if your scaler needs to connect to an external source based on a connection string, you expect the user to supply this connection string in the confguration within the ScaledObject under `trigger`. This data will be passed to your constructing function as map[string]string.
```

Comment:
```suggestion
2. Define the main pieces of data that you expect the user to supply so the scaler runs properly. For example, if your scaler needs to connect to an external source based on a connection string, you expect the user to supply this connection string in the configuration within the ScaledObject under `trigger`. This data will be passed to your constructing function as map[string]string.
```

Reply:
oh lol i fixed a typo with another typo! 🤦‍♂

Reply:
No worries, that's what reviews are for!